### PR TITLE
prep2GADS raw

### PR DESCRIPTION
**Pull request by sachseka**
Please review the latest changes at your earliest convenience. This task is of medium priority and has no immediate urgency.

### CODE REVIEW

**Review by franikowsp — COMMENTED** [review]
Sorry again for the extreme delay. 🙈 Only some, mostly tiny, nits - looks to me like a very good extension for the newly supported `trafoType`!

File: R/prep2gads.R [hunk 1]
```diff
@@ -12,74 +12,111 @@ prep2GADS <- function (dat, inputList, trafoType = c("scored", "raw"),
...
 
   labels2 <- dplyr::full_join(values3, labels1, by="varName")
   labels2 <- labels2[,c("varName", "varLabel", "format", "display_width", "labeled", "value", "valLabel", "missings")]
   labels2$labeled <- ifelse(!is.na(labels2$varLabel) | !is.na(labels2$valLabel), "yes", "no")
-  suppressWarnings(labels2 <- collapseMissings(labels2, missing.rule = misTypes, standard=FALSE))
 
-  dat2 <- collapseMissings(dat, missing.rule = misTypes, standard=FALSE)
+  if(trafoType=="scored") {
+    suppressWarnings(labels2 <- collapseMissings(labels2, missing.rule = misTypes, standard=FALSE))
+    dat2 <- collapseMissings(dat, missing.rule = misTypes, standard=FALSE)
```
[the remark this answers is not in the document]
Comment (sachseka) [reply R/prep2gads.R:116]: To change this I would have to add the same lines to trafoType="raw" and I prefer it more efficient like it is...

[the remark this answers is not in the document]
Comment (franikowsp) [reply R/prep2gads.R:116]: Sorry, the markup was off 🤯: I only wanted to address this line with my comment as it has no dependencies outside of the first large if-else statement:

```r
    dat2 <- collapseMissings(dat, missing.rule = misTypes, standard=FALSE)
```

**Comment by sachseka** [comment]
@franikowsp Great thanks! I followed *all* your NITs :)

**Pull request merged**